server.js

```diff
@@ -1,60 +1,60 @@
-// TASK 1: Setting up server
+// // TASK 1: Setting up server
 
 
-const http = require('http');
-
-const server = http.createServer((req, res) => {
-  res.statusCode = 200;
-  res.setHeader('Content-Type', 'text/plain');
-  res.end('Hello World\n');
-});
+// const http = require('http');
 
-const PORT = 80;
-const HOSTNAME = '0.0.0.0';
+// const server = http.createServer((req, res) => {
+//   res.statusCode = 200;
+//   res.setHeader('Content-Type', 'text/plain');
```

Comment:
The mongoUri variable should not be hardcoded in the code. It should be stored in a separate configuration file.